rosegarden-appdata-id-and-desktop-launchable.patch

```diff
@@ -0,0 +1,12 @@
+--- rosegarden-1/data/appdata/rosegarden.appdata.xml	2024-06-07 18:27:30.117123029 +0100
++++ rosegarden-1/data/appdata/rosegarden.appdata.xml	2024-06-07 18:29:04.831009405 +0100
+@@ -1,7 +1,8 @@
+ <?xml version="1.0" encoding="utf-8"?>
+ <!-- Copyright 2018-2024 Rosegarden development team -->
+ <component type="desktop">
+-    <id>com.rosegardenmusic.rosegarden.desktop</id>
++    <id>com.rosegardenmusic.rosegarden</id>
```

Comment:
this requires to have a `provides` for the old name. `flatpak-builder` fix this automatically.

Reply:
Would you like me to undo the `id` change? If so I'm happy to do it.

Reply:
no. add the provides.

Reply:
actually yes you can just revert this since the appstream file is renamed.

Reply:
but this should be addressed upstream regardless. It's illogical the name they have upstream.

Reply:
> actually yes you can just revert this since the appstream file is renamed.

I took care of that [in this recently pushed commit](https://github.com/flathub/com.rosegardenmusic.rosegarden/pull/15/commits/bcee81dd3e0235c43441adec46efbfa4a487372d) :+1: